README.md

```diff
@@ -216,6 +224,8 @@ type TestType = typeof Test;
 
 ### TypeScript
 
+In Typescript, classes themself are types, so you don't need to define a type for them. But, if you must, you can do it by:
```

Comment:
This is true also for Flow. Perhaps it would be good as a remark about both of these systems, not specifically in the `TypeScript` section.

Reply:
I didn't know this. I will update this part.